rlp/rlpgen/testdata/alias.in.txt

```diff
@@ -0,0 +1,22 @@
+// -*- mode: go -*-
+
+package test
+
+import (
+    "math/big"
+    "github.com/holiman/uint256"
+)
+
+// Alias types chosen because their originals have special handling that is easy
+// to spot when inspecting generated output.
+type (
+    Big = big.Int
+    // Demonstrate recursive unaliasing
```

Comment:
this is also to demonstrate types.Unalias is a better fit than .Underlying() I suppose, right (especially the uint256.Int)?

Reply:
Nah `Underlying()` wouldn't work even without the recursion. This is just being thorough.